src/ActorMultiVertex.cpp

```diff
@@ -160,46 +160,60 @@ void ActorMultiVertex::UnloadTexture()
 
 void ActorMultiVertex::SetNumVertices( std::size_t n )
 {
-	if( n == 0 )
-	{
-		for( std::size_t i = 0; i < AMV_Tweens.size(); ++i )
-		{
-			AMV_Tweens[i].vertices.clear();
-		}
-		AMV_current.vertices.clear();
-		AMV_start.vertices.clear();
-	}
-	else
-	{
-		for( std::size_t i = 0; i < AMV_Tweens.size(); ++i )
+    for( std::size_t i = 0; i < AMV_Tweens.size(); ++i )
+    {
+        if( n == 0 )
+        {
+            AMV_Tweens[i].vertices.clear();
+        }
+        else
+        {
+            AMV_Tweens[i].vertices.resize( n );
+        }
+    }
+
+    if( n == 0 )
+    {
+        AMV_current.vertices.clear();
+        AMV_start.vertices.clear();
+    }
+    else
+    {
+        AMV_current.vertices.resize( n );
+        AMV_start.vertices.resize( n );
+    }
+}
+
+void ActorMultiVertex::ResizeVertices(std::vector<RageSpriteVertex>& vertices, int size)
+{
+	static RageSpriteVertex newVertex;
+	if (vertices.capacity() < size)
 		{
-			AMV_Tweens[i].vertices.resize( n );
+		vertices.reserve(size);
```

Comment:
I think so because the idea is that the game has a moment to allocate the memory instead of having to interrupt the game more suddenly. Most of our issues regarding stutter or weird issues like that seem to be traced back to memory reallocations of miniscule amounts. At least on Windows with the build flags used for MSVC, this does seem to make a difference.